TemporaryParser/TPRBComment.class.st

```diff
@@ -28,13 +28,28 @@ Class {
 	#superclass : #TPRBNode,
 	#instVars : [
 		'contents',
-		'start'
+		'start',
+		'stop'
 	],
 	#category : #'TemporaryParser-Nodes'
 }
 
+{ #category : #'instance creation' }
+TPRBComment class >> with: aCommentToken [
+	| str cpystr |
+	str := aCommentToken value.

```

Comment:
Say here that the quotes are being stripped.

Also, the variable `str` can be called `commentString` and `cpystr` into `commentQuotes`